demos/wasi/hello-world-rust/k8s.yaml

```diff
@@ -28,18 +13,13 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-        - name: CONFIG_MAP_VAL
-          valueFrom:
-            configMapKeyRef:
-              key: myval
-              name: hello-world-wasi-rust
       volumeMounts:
         - name: storage
           mountPath: /mnt/storage
   volumes:
     - name: storage
-      configMap:
```

Comment:
We should probably do a separate example using the same hello world but with PVC

Reply:
agreed. Might need to point to the documentation on setting up the example too, but yes.